openlibrary/views/loanstats.py

```diff
@@ -96,3 +98,47 @@ def is_enabled(self):
     def GET(self):
         stats = WLStats()
         return app.render_template("stats/waitinglists.html", stats)
+
+class readinglog_stats(app.view):
+    path = "/stats/readinglog"
+
+    def GET(self):
+        ONE_MONTH_DATE = datetime.date.today() - datetime.timedelta(days=28)
```

Comment:
I think we need to be able to specify the date span.

Reply:
This is probably right -- at this point I'd prefer opening a separate issue and supporting it as its needed. The primary use case for this feature was/is internal weekly metrics updates (which we want the public to have access to as well). If anyone feels querying by date is a blocker, I'm happy to spend time reviewing a pull request which addresses that.